### PR TITLE
Adding support for package management to pyglidein

### PR DESCRIPTION
**Pull request by hskarlupka**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    author='WIPAC',
+    author_email='contact-us@icecube.wisc.edu',
+    classifiers=[
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+    ],
```
Comment (dsschult) [inline setup.py]: What about python 3.6? :)

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    keywords='htcondor htc glidein',
```
Comment (dsschult) [inline setup.py]: Is this htc, or dhtc?  Maybe better to just say "distributed computing"

Comment (briedel) [reply setup.py]: dHTC

File: setup.py (hunk 1 above)
Comment (briedel) [reply setup.py]: And what about 2.6? Unless we assume we run parrot, cvmfs, or modules everywhere

Comment (dsschult) [reply setup.py]: @briedel noticed that 2.6 isn't listed.  We probably need to continue support for that until we stop running on SL6.

Comment (hskarlupka) [reply setup.py]: @briedel @dsschult I confirmed that Minio completely fails on 2.6.  I'm going to so something like:

`'minio;python_version>"2.7"',` in the setup.py file
and add conditional imports of minio in the client_util module.  

For now log shipping on python 2.6 will just have to be disabled.

File: setup.py (hunk 2 above)
Comment (hskarlupka) [reply setup.py]: Updating to dHTC.

**Comment by hskarlupka** [comment]
close #114

**Comment by hskarlupka** [comment]
close #114

**Pull request merged**